apps/sms/test/marionette/inbox_test.js

```diff
@@ -69,7 +69,6 @@ marionette('Inbox View tests', function() {
       newMessageView.assertRecipientsInputFocused();
 
       // We should enter new message view even if not all messages are rendered.
-      assert.isTrue(inboxView.hasConversation(10));
```

Comment:
It was basically useless check and as it turned out - not reliable as well - when environment is slow like on the Treeherder there is a chance that we can navigate to Compose view even before 10th conversation is rendered,